README.md

```diff
@@ -66,27 +80,30 @@ yarn start
 
 Visit your app on: `http://localhost:3000`. You can interact with your smart contract using the contract component or the example ui in the frontend. You can tweak the app config in `packages/nextjs/scaffold.config.ts`.
 
-Run smart contract test with `yarn hardhat:test`
+Run smart contract test with `yarn hardhat:test` or `yarn foundry:test` depending of your solidity framework.
```

Comment:
This is more a dev comment (rather than a doc comment), but should we rename all these scripts as `yarn chain:*` so they don't change with the framework selected?

cc @carletex